### PR TITLE
Wallabag - new role

### PR DESCRIPTION
**Pull request by addbee**
Only just started using this but seems to be a common request with other combination packages.

Using now to evaluate the stability - if others want to try it before merging, feel free.

### CODE REVIEW

**Comment by addbee** [comment]
I've forked the cloudbox_mod template and amended to run independtly - so if interest builds it could be merged with Community, but for now it can be used at:
https://github.com/addbee/cloudbox_wallabag

**Comment by desimaniac** [comment]
I have no issues with merging it if it works.. just a had a question about the ports. Does 7780 need to be exposed to the outside ? Can it not be handles via reverse proxy ?

**Comment by addbee** [comment]
Yes- I agree that the port shouldn't be exposed. 

I included it because it seemed to be a common pattern on the other roles to have an internal port exposed - i didn't see the need most of the time.

**Comment by desimaniac** [comment]
They hide it being 127.0.0.1. Which limits access from the local machine only. I’m getting rid of that as well in coming CB release.

File: roles/wallabag/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      PUID: "{{ uid }}"
+      PGID: "{{ gid }}"
+      PASSWORD: "{{ user.pass }}"
+      VIRTUAL_HOST: "bag.{{ user.domain }}"
+      VIRTUAL_PORT: "80"
+      LETSENCRYPT_HOST: "bag.{{ user.domain }}"
+      LETSENCRYPT_EMAIL: "{{ user.email }}"
+      SYMFONY__ENV__DOMAIN_NAME: "https://bag.{{ user.domain }}"
+    ports:
+      - "7780:80"
```
Comment (Migz93) [inline roles/wallabag/tasks/main.yml]: Change this line so it doesn't expose the port externally. Something like this
      - "127.0.0.1:7780:80"

Once changed I can test this before we merge

Comment (addbee) [reply roles/wallabag/tasks/main.yml]: Done

**Comment by Migz93** [comment]
addbee made changes so that it is only exposed via the reverse proxy as Desi requested.

Installed and tested this PR, confirmed it works as expected, merging.

**Pull request merged**